README.md

```diff
@@ -1,181 +1,248 @@
+[![Discord chat][discord-badge]][discord-url]
+[![Playground][playground-badge]][playground-url]
 </div>
 
-[discord-badge]: https://img.shields.io/discord/1079625926024900739?logo=discord&label=discord&color=brightgreen
-[discord-url]: https://discord.gg/9uXCAwqQZW
-[mit-badge]: https://img.shields.io/badge/license-MIT-blue.svg?color=brightgreen
-[mit-url]: LICENSE
-[ci-badge]: https://github.com/web-infra-dev/oxc/actions/workflows/ci.yml/badge.svg?event=push&branch=main
-[ci-url]: https://github.com/web-infra-dev/oxc/actions/workflows/ci.yml?query=event%3Apush+branch%3Amain
-[npm-badge]: https://img.shields.io/npm/v/oxlint/latest?color=brightgreen
-[npm-url]: https://www.npmjs.com/package/oxlint/v/latest
-[crates-badge]: https://img.shields.io/crates/v/oxc_parser.svg
-[crates-url]: https://crates.io/crates/oxc_parser
-[docs-badge]: https://docs.rs/oxc/badge.svg
-[docs-url]: https://docs.rs/oxc
-[code-size-badge]: https://img.shields.io/github/languages/code-size/web-infra-dev/oxc
-[code-size-url]: https://github.com/web-infra-dev/oxc
-[code-coverage-badge]: https://codecov.io/github/web-infra-dev/oxc/branch/main/graph/badge.svg
-[code-coverage-url]: https://codecov.io/gh/web-infra-dev/oxc
-
-The Oxidation Compiler is creating a suite of high-performance tools for the JavaScript / TypeScript language re-written in Rust:
+## ⚓ Oxc
 
-* [AST](./crates/oxc_ast) - See [docs.rs][docs-ast]
-* [Parser](./crates/oxc_parser) ([acorn][acorn]) - See [docs.rs][docs-parser], [@oxidation-compiler/napi][npm-napi]
-* [Linter](./crates/oxc_linter) ([ESLint][ESLint]) - Prototype - Try it out! `npx oxlint@latest path`
-* Formatter ([prettier][prettier])
-* Transpiler ([babel][babel])
-* [Minifier](./crates/oxc_minifier) ([terser][terser]) - [Prototype](https://github.com/web-infra-dev/oxc/tree/main/crates/oxc_minifier)
-* [Resolver](./crates/oxc_resolver) ([enhanced-resolve](enhanced-resolve))
-* Type Checker - See [ezno][ezno], available via `npx oxidation-compiler@latest check path`
+The Oxidation Compiler is creating a suite of high-performance tools for JavaScript and TypeScript.
 
-[docs-ast]: https://docs.rs/oxc/latest/oxc/ast/index.html
-[docs-parser]: https://docs.rs/oxc/latest/oxc/parser/index.html
-[npm-napi]: https://www.npmjs.com/package/@oxidation-compiler/napi
-[acorn]: https://github.com/acornjs/acorn
-[babel]: https://babel.dev
-[prettier]: https://prettier.io
-[ESLint]: https://eslint.org/
-[prettier]: https://github.com/prettier/prettier
-[ezno]: https://github.com/kaleidawave/ezno
-[terser]: https://terser.org
-[enhanced-resolve]: https://github.com/webpack/enhanced-resolve
+Oxc is a parser, linter, formatter, transpiler, minifier, resolver ... all written in Rust.
 
-## Philosophy
+## 💡 Philosophy
 
-This project follows philosophies from the [Rome](https://rome.tools) and [Ruff](https://beta.ruff.rs) projects.
+This project adheres to philosophies from [Rome][rome] and [Ruff][ruff].
```

Comment:
Does this mean that if their philosophies change, you'll change yours?
I recommend that you mark those philosophies "as yours" and mention Rome and Ruff as references.

Reply:
We inherited from Rome, Rome will never change, so I guess we are safe from Rome ever changing their philosophy 😅